### ISSUE TITLE
Call for Contributors

### ISSUE THREAD

**Issue by caleblloyd** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#522]
`Pomelo.EntityFrameworkCore.MySql` is looking for core contributors to help with to help with maintenance and upgrades for the MySql Entity Framework Core Driver to track [EF Core Upstream Version 2.1](https://github.com/aspnet/entityframeworkcore)  and beyond (see issue #514)

I have worked on the previous few minor and major upgrades, and they can take a fair amount of work.  I am no longer using EF at my day job and do not have as much capacity as in times past to contribute.  I can say that it is very fun and rewarding work, and that the EF Core team at Microsoft are amazing folks and responsive when I get completely stuck.  I'm glad to still participate in code reviews and may try to contribute more as I find time.

EF Core performs upstream major, minor, or patch releases that can require re-work of provider code approximately every 6 months.  An ideal core contributor would use both Entity Framework Core and MySQL often and have the time to spend the time interfacing with the EF Upstream team to understand upcoming changes.  That being said, all contributions are welcome.

Thanks for everyone's support over the 1.5 years that I've been working on the project, I look forward to seeing what the community continues to do!

cc @yukozh @mguinness

**Comment by ngochivinh** [comment]
Hi @caleblloyd ,
I'm a software engineer, I want to be a contributor to this project. How can I join the project?
Thank you.
email: vnngochivinh@gmail.com

**Comment by mguinness** [comment]
@caleblloyd @yukozh Does joining [Open Collective](https://opencollective.com/faq) make any sense for this project?  It could be a way to compensate contributors for their time and effort working on the project.

They let any open source project with at least 100 stars on GitHub join under their umbrella where they will act as a host organization so that you can start collecting donations (both one-off and recurring) and paying expenses without the headache of setting up a 501(c)(3) and a formal structure. You just add a bunch of trusted admins and you can manage expenses and invoices. Open Collective holds the money for you.

**Comment by caleblloyd** [comment]
> Does joining Open Collective make any sense for this project?

I don't think it makes a lot of sense at this point, there are currently no expenses associated with this project that I know of.  GitHub, TravisCI, AppVeyor, and MyGet are all free for open source projects.  I like the idea of contributors supporting the project either as personal contributions or companies contributing developer time if their business depends on the project.

**Comment by VictorHaine** [comment]
I'm starting to use Pomelo.EF.MySql and when I get on track maybe I can contribute. Let's see!

**Comment by T-rav** [comment]
I would like to help with the 2.1 migration but...
When I pull the f_2.1.0-preview branch and run locally it has 36 failing test. All of the failures are because of - "Message: System.InvalidOperationException : The property 'GeneratedTime.CreatedTimestamp' could not be mapped, because it is of type 'DateTime' which is not a supported primitive type or a valid entity type. "

Is this normal? I have dotnet core 2.1.300-rc1-008673 installed.

**Comment by T-rav** [comment]
Thanks, but even more test are failing on that branch. I am so not have
confidence to make changes when code is broken.

I guess I will pick this up once the test are all green.

**Comment by caleblloyd** [comment]
We are going to have to ship the first RC with most functional tests turned off in CI due to failing tests in #552   It would be very helpful for contributors to work through those test failures, help find fixes, and open PRs.

To run all Functional Tests:
```
dotnet test -f netcoreapp2.1 -c Release test/EFCore.MySql.FunctionalTests
```

To run a single Functional Tests in order to zero-in on an issue:
```
dotnet test -f netcoreapp2.1 -c Release test/EFCore.MySql.FunctionalTests --filter "FullyQualifiedName=Namespace.ClassName.MethodName"
// example:
dotnet test -f netcoreapp2.1 -c Release test/EFCore.MySql.FunctionalTests --filter "FullyQualifiedName=Pomelo.EntityFrameworkCore.MySql.FunctionalTests.Scaffolding.MySqlDatabaseModelFactoryTest.Set_unique_for_unique_index"
```

**Comment by IAMRogerXi** [comment]
@caleblloyd  do you still need the contributor for the project Pomelo.EntityFrameworkCore.MySql? thanks.

**Comment by mguinness** [comment]
@IAMRogerXi We're certainly still looking for additional contributors.  To get started, take a look at [help wanted](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/issues?q=is%3Aissue+is%3Aopen+label%3Ahelp-wanted) issues.

**Comment by mguinness** [comment]
@IAMRogerXi If you are still interested in contributing you could take a look at https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/issues/698.

**Comment by IAMRogerXi** [comment]
Sure. However, I notice the issue is closed.?

**Comment by DavisSylvester** [comment]
@caleblloyd Are you still looking for contributors?  I am willing to assist. if help is still required

**Comment by caleblloyd** [comment]
@DavisSylvester yes we are!  A good way to start is to fork the repository, then work on a branch to fix an open issue or feature. Once it is ready, open a pull request back to this repository and we will do a code review.

There will be a big lift that we need help with once EF Core Upstream 3.0.0 is published.

**Comment by Mousavi310** [comment]
Hi, how can we see upstream changes of Microsoft ef core team? Are them public? Or should we read the ef core repository changes?
